static/app/gettingStartedDocs/node/node.tsx

```diff
@@ -196,6 +196,10 @@ Sentry.init({
   // of transactions for performance monitoring.
   // We recommend adjusting this value in production
   tracesSampleRate: 1.0,
+
```

Comment:
don't we want to display this for the other node platforms?

![image](https://github.com/user-attachments/assets/f3c77d1c-a0d2-42f3-b3a3-d8301db19cb9)

Reply:
it already is for all of them, because they do use some shared snippet like you suggested for the browser platforms :D . Nest.js for example

![image](https://github.com/user-attachments/assets/16be7c63-cb37-4ed8-8f34-be2ba475b213)